src/cmds/qsub.c

```diff
@@ -3140,14 +3154,14 @@ set_opt_defaults(void)
 static int
 get_script(FILE *file, char *script, char *prefix)
 {
-	char s[MAX_LINE_LEN + 1];
 	char *sopt;
 	int err = 0;
 	int exec = FALSE;
-	char *cont;
 	char tmp_name[MAXPATHLEN + 1];
 	FILE *TMP_FILE;
 	char *in;
+	char *s_in;
```

Comment:
To be on he safer side, I think you should set s_in to NULL here, so that just in case pbs_fgets_extend fails, you don't end up freeing garbage memory with free(s_in) later.

Reply:
This is a remote possibility, but it is a possibility nonetheless. I have addressed your comment Ravi, please have a look.